### PR TITLE
Add CONTRIBUTORS and LICENSE files

### PR DESCRIPTION
**Pull request by bearbin**
Requires sign-off by copyright holders.

I noticed @fankins and @agusnat16 having committed several times. If their contributions are still evident they will need to sign off also.

### CODE REVIEW

**Comment by bearbin** [comment]
This fixes #7.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,208 @@
+Copyright 2013-2016 Cuberite Contributors
```
Comment (NiLSPACE) [inline LICENSE]: Seems like the year isn't correct.

Comment (bearbin) [inline LICENSE]: What should the year be?

Comment (NiLSPACE) [inline LICENSE]: 2017?

Comment (bearbin) [inline LICENSE]: Nope, no commits of any substance since jan 2016. Since nothing has changed, the copyright date can't change either.

Comment (madmaxoft) [inline LICENSE]: I hate legal stuff

**Comment by madmaxoft** [comment]
The original code by @fankins and @agusnat16 has been completely replaced, so I don't think their explicit consent is required anymore.

File: CONTRIBUTORS [hunk 2]
```diff
@@ -0,0 +1,11 @@
+This file contains all known copyright holders of this software, as far as is
+practically possible to ascertain.
+
+If you contribute to this software you must add yourself to this file, to
+indicate your agreement to license your contributions according to the license
+as provided in the LICENSE file.
+
+mathiascode
+NiLSPACE
+SphinxC0re (Julian Laubstein)
```
Comment (sphinxc0re) [inline CONTRIBUTORS]: it's `sphinxc0re`

Comment (sphinxc0re) [inline CONTRIBUTORS]: it's `sphinxc0re`

Comment (bearbin) [inline CONTRIBUTORS]: Fixed.

**Comment by bearbin** [comment]
@sphinxc0re, @mathiascode Can I get your approval?

**Pull request merged**